connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/functions.scala

```diff
@@ -36,7 +36,8 @@ object functions {
    * @param messageName
    *   the protobuf message name to look for in descriptorFile.
    * @param descFilePath
-   *   the protobuf descriptor in Message GeneratedMessageV3 format.
+   *   the protobuf descriptor file.
```

Comment:
Removed `GeneratedMessageV3` from descriptions in multiple places. It matters only for Java classes. In addition we support both V2 and V3 protobuf syntax.